WcaOnRails/config/routes.rb

```diff
@@ -42,6 +42,11 @@
     get 'register' => 'registrations#register'
     get 'register-require-sign-in' => 'registrations#register_require_sign_in'
   end
+
+  get 'competitions/:competition_id/report/edit' => 'delegate_reports#edit', as: :delegate_report_edit
+  get 'competitions/:competition_id/report' => 'delegate_reports#show', as: :delegate_report
+  patch 'competitions/:competition_id/report' => 'delegate_reports#update'
```

Comment:
Actually I thought just about some cleanup like
```ruby
resources :delegate_reports, path: "competitions/:competition_id/report", only: [:edit, :update, :show]
```
(I'm not even sure whether this works but that was my first thought, if it doesn't maybe it's not worh digging into it)